packages/block-editor/src/store/selectors.js

```diff
@@ -1537,3 +1537,14 @@ export function didAutomaticChange( state ) {
 export function __unstableGetSelectedMountedBlock( state ) {
 	return state.selectedMountedBlock;
 }
+
+/**
+ * Returns the current editing canvas device type.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {string} Device type.
+ */
+export function deviceType( state ) {
```

Comment:
I think this should be `getDeviceType` so that it matches the naming convention of our other selectors.